geode-core/src/main/java/org/apache/geode/management/internal/api/LocatorClusterManagementService.java

```diff
@@ -653,7 +651,23 @@ public void close() {
         }
       }
     }
+    return results;
+  }
 
+  @VisibleForTesting
+  <R> List<R> cleanResults(List<?> functionResults) {
+    List<R> results = new ArrayList<>();
+    for (Object functionResult : functionResults) {
+      if (functionResult == null) {
+        continue;
+      }
+      if (functionResult instanceof Throwable) {
+        // log the exception and continue
+        logger.info("Error executing CacheRealizationFunction.", (Throwable) functionResult);
```

Comment:
A `warn` level maybe more appropriate than `info` level for Throwable. Some more information in the log might help the user to diagnose in case of an error. e.g. Function name etc. I would probably consider logging the stack trace.